docs/quickstart.md

```diff
@@ -1,428 +1,256 @@
-        default:
-          description: unexpected error
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/Error"
-    post:
-      summary: Create a pet
-      operationId: createPets
-      tags:
-        - pets
-      responses:
-        201:
-          description: Null response
-        default:
-          description: unexpected error
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/Error"
-  /pets/{petId}:
-    get:
-      summary: Info for a specific pet
-      operationId: showPetById
-      tags:
-        - pets
-      parameters:
-        - name: petId
-          in: path
-          required: true
-          description: The id of the pet to retrieve
-          schema:
-            type: string
-      responses:
-        200:
-          description: Expected response to a valid request
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/Pets"
-        default:
-          description: unexpected error
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/Error"
-components:
-  schemas:
-    Pet:
-      required:
-        - id
-        - name
-      properties:
-        id:
-          type: integer
-          format: int64
-        name:
-          type: string
-        tag:
-          type: string
-    Pets:
-      type: array
-      items:
-        $ref: "#/components/schemas/Pet"
-    Error:
-      required:
-        - code
-        - message
-      properties:
-        code:
-          type: integer
-          format: int32
-        message:
-          type: string
-```
+Go [here](openapi-starter.md) to learn how to clone the `openapi-starter` project and learn more about how it works.
+
+## Step 2 - Install OpenAPI CLI from openapi-starter
+
+Run `npm install` in the root of the `openapi-starter` repo.
+
+## Step 3 - Prepare your API definition file
+
+In this guide, we'll use `openapi.yaml` in all examples. We recommend that you use it until you're confident with OpenAPI CLI.
+
+## Step 4 - Try some basic commands
+[Commands](./docs/commands/index.md) are used to run tasks (like splitting up large definition files and putting them back together) and they can also return information (like getting stats about your definition). To get you started, we'll focus on the top four: `bundle`, `lint`, `split` and `preview`.
 
-## Step 2 - Try basic commands
+:::success Tip
+If you're new to command lines, you don't have to include the `$` in your commands. Just open the node.js command window, make sure you `cd` to the path where your API definition file is stored, type the command then press **Enter**.
+:::
 
-### stats - get a short summary about your definition files
+### `bundle` - Pull constituent parts of your definition back into a single file
```

Comment:
issue: This heading is quite long - I think it's the longest of all on this page (the previous one had the same problem). This might cause issues with the table of contents on the right side of the page. It won't necessarily break anything, but it will look awkward. Do you have any ideas for how we could shorten it? If not, it's fine, not a blocker - we can leave it and maybe get back to it later.